e2e/cypress/support/api/user.js

```diff
@@ -35,14 +35,18 @@ Cypress.Commands.add('apiLogout', () => {
         log: false,
     });
 
-    // Ensure we clear out these specific cookies
+    // * Verify logged out
+    cy.visit('/login?extra=expired').url().should('include', '/login?extra=expired');
+
```

Comment:
Since this is a generic function and all the tests using this will be affected by this. Can we include this step specifically only in the forgot password spec?

Reply:
The problem is that step is in between the logout itself and deletion of cookies. Not sure how to separate that for forgot password spec. Or do you suggest I recreate a new command just for forgot password?

Reply:
The purpose is to verify user is logged out before cookies are deleted